pkg/collector/upgrade/upgrade.go

```diff
@@ -55,6 +55,12 @@ func (u VersionUpgrade) ManagedInstances(ctx context.Context) error {
 	for i := range list.Items {
 		original := list.Items[i]
 		itemLogger := u.Log.WithValues("name", original.Name, "namespace", original.Namespace)
+
+		if original.Spec.ManagementState == v1alpha1.ManagementStateUnmanaged {
```

Comment:
We should book a ticket to run the upgrade as part of the reconciliation for instances that will be switched from unmanaged to managed after the upgrade

Reply:
> We should book a ticket to run the upgrade as part of the reconciliation for instances that will be switched from unmanaged to managed after the upgrade

#1890